### DISCUSSION TITLE
Add HCP Packer support

### DISCUSSION THREAD

**Discussion by sestegra** [vmware/packer-examples-for-vsphere#230] [Ideas]
A configuration parameter should be added to support HCP Packer.
What do you think about it?

```
build {
  dynamic "hcp_packer_registry" {
    for_each = var.common_hcp_packer_enabled == true ? [1] : []
    content {
      ..
    }
  }
}
```

Reference
https://cloud.hashicorp.com/products/packer

**Comment by tenthirtyam** [comment]
Yes! It's been sitting on my list planned additions alongside some CI after the recent OS and minor enhancements.

I've been meaning to make this one happen using a similar approach as the content library and export with the dynamic block. 

I'll convert this one over to an enhancement issue and we can collaborate on it. If we are able to do so early this week and test it we can skip it into 22.07 or hold for 22.08.

**Ryan Johnson**
Senior Staff Solutions Architect | Product Engineering @ VMware, Inc.
